qa/tasks/cephfs/fuse_mount.py

```diff
@@ -312,7 +312,9 @@ def umount(self, cleanup=True):
         umount() must not run cleanup() when it's called by umount_wait()
         since "run.wait([self.fuse_daemon], timeout)" would hang otherwise.
         """
-        if not self.is_mounted():
+        if not self.is_mounted() or self.is_stuck():
+            if self.is_stuck():
```

Comment:
If `self._run_umount_lf()` is dependent on if `is_stuck()` is true or not, can't we just do `if self.is_stuck(): self._run_umount_lf()` straightaway instead of the current approach?